serverless/telegram-handler/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { parseEvent } from './parser'
+import sequelizeLoader from './sequelize-loader'
+import { getBotTokenFromId, verifyBotIdRegistered } from './credentials'
+import { handleUpdate } from './bot'
+
+const handler = async (event: any): Promise<{ statusCode: number }> => {
+  try {
+    // Parse botId and Telegram update
+    const { botId, update } = parseEvent(event)
+
+    // Verify botId and fetch bot token
+    const sequelize = await sequelizeLoader()
```

Comment:
If you define this outside of the handler, you can reuse the warm connection for multiple invocations of the lambda.